bqplot/marks.py

```diff
@@ -858,7 +858,7 @@ class Bars(Mark):
         Y respectively.
     type: {'stacked', 'grouped'}
         whether 2-dimensional bar charts should appear grouped or stacked.
-    colors: list of colors (default: CATEGORY10)
+    colors: list of colors (default: ['steelblue'])
```

Comment:
For a 1-D `Bar`  to have the same color for each bar, you can change the default value of the attribute `color_mode`. I prefer that rather than having just one default color.